src/utils/lock.ts

```diff
@@ -57,7 +57,6 @@ export const createWithdrawTx = (chainId: string) => {
     to: CHAIN_SAFE_LOCKING_ADDRESS[chainId],
     data: safeLockingInterface.encodeFunctionData('withdraw', [0]),
     value: '0',
-    operation: 0,
```

Comment:
Why do we not need this anymore?

Reply:
We never needed it.
You cannot do a DELEGATE CALL through our safe apps communicator. So this value was always ignored anyways.